pkg/minikube/cruntime/docker.go

```diff
@@ -106,6 +106,15 @@ func (r *Docker) SocketPath() string {
 
 // Available returns an error if it is not possible to use this runtime on a host
 func (r *Docker) Available() error {
+	// If Kubernetes version >= 1.24, require both cri-dockerd and dockerd.
+	if r.KubernetesVersion.GTE(semver.Version{Major: 1, Minor: 24}) {
+		if _, err := exec.LookPath("cri-dockerd"); err != nil {
+			return err
```

Comment:
nit pick but I suggest wrapping these two errors

Reply:
Yes, adding in an error template with installation instructions.